security/build.gradle.kts

```diff
@@ -10,9 +10,11 @@ dependencies {
     api(mnSerde.micronaut.serde.api) {
         exclude(group = "io.micronaut", module = "micronaut-json-core")
     }
-    api(mnValidation.validation)
+    api(mnValidation.validation) //  // jakarta.validation:jakarta.validation-api
```

Comment:
What's the comment for here?

Reply:
it is totally alien to me that `mnValidation.validation` means `jakarta.validation:jakarta.validation-api`.  If instead of`mnValidation.validation` it was `mnValidation.jakarta.validation.api`I would have not added the comment.